src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -133,18 +136,45 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
 
     with self.argument_context('ams content-key-policy') as c:
         c.argument('account_name', account_name_arg_type)
-        c.argument('content_key_policy_name', name_arg_type)
+        c.argument('content_key_policy_name', name_arg_type,
+                   help='The content key policy name.')
         c.argument('description', help='The content key policy description.')
         c.argument('clear_key_configuration',
                    action='store_true',
-                   arg_group='Basic Policy Options',
+                   arg_group='Clear Key Configuration (AES Encryption)',
                    help='Use Clear Key configuration, a.k.a AES encryption. It\'s intended for non-DRM keys.')
         c.argument('open_restriction',
                    action='store_true',
-                   arg_group='Basic Policy Options',
-                   help='Use open restriction. License or key will be delivered on every request.')
-        c.argument('policy_option_name',
-                   help='The name of the policy option.')
+                   arg_group='Open Restriction',
+                   help='Use open restriction. License or key will be delivered on every request. Not recommended for production environments.')
+        c.argument('policy_option_name', help='The content key policy option name.')
+        c.argument('policy_option_id', help='The content key policy option identifier.')
```

Comment:
Add something like `You can get this value from 'policyOptionId' property by running a show operation on this resource`